doc/radosgw/external-authentication.rst

```diff
@@ -0,0 +1,101 @@
+========================
+ External Authentication
+========================
+
+It is possible to use a custom authentication server for Ceph Object Gateway.
+With this option you can use your own authentication server to authenticate
+users with all RGW User features like SubUser, Tenant, Permissions.
+The only thing you should obey is to implement these APIs so we can get needed
+data from your sever.
+
```

Comment:
Suggest something like

It is possible to use an external authentication server for the Ceph Object Gateway (RGW), providing 
all user features including subuser, tenant, and permissions.  The external server must implement that API as below.